manifest.json

```diff
@@ -2264,6 +2264,28 @@
                 }
             }
         },
+        "net.eia485.ArrowExpanders": {
+            "name": "ArrowExpanders",
+            "description": "keybinds for navigating and interacting with expander tress",
```

Comment:
tress vs trees, Also could include or add a tag for inspectors as that'd be the main usage